conference/views/__init__.py

```diff
@@ -240,8 +235,7 @@ def schedule_events_booking_status(request, conference):
         else:
             v['user'] = False
         del v['booked']
-    return dat
-    a
```

Comment:
wut? 😆

Reply:
I know :laughing: must have been a typo introduced at some point, and we're probably not using this view anymore - another candidate to retire.